src/frontend/components/VideoPlayer/index.tsx

```diff
@@ -136,6 +143,14 @@ const VideoPlayer = ({
           ))}
       </video>
       {!player && video.live_state && <WaitingLiveVideo />}
+      {player &&
+        video.live_state &&
+        [liveState.STOPPING, liveState.PAUSED].includes(video.live_state) && (
```

Comment:
this [comment](https://github.com/openfun/marsha/pull/1216/files#r753176160) may apply here too